src/mapping/mapper.js

```diff
@@ -0,0 +1,182 @@
+const fhirpath = require('fhirpath');
+const _ = require("underscore");
+
+let isTrue = (arg) => {
+  if(Array.isArray(arg) ){
+    return arg.find(i => isTrue(i));
+  }else if(typeof arg === "object"){
+    return !_.isEmpty(arg);
+  }else if(typeof arg === "string" && arg ==="false"){
+    return false;
+  }
+  return arg;
+}
+
+let buildProcessor = (arg) => {
```

Comment:
I'm not clear on why this one is called "buildProcessor" since the resulting function is only used as a "default function" from what I can see. Plus this function if given a string does a `require` which is a little unexpected... Can you elaborate on what this function and the "default" functions are for?

Reply:
@dehall I updated the code and added comments to make it clearer as to what was attempting to be accomplished